### PR TITLE
chore: remove from client check if browser is co-located with server

### PR DESCRIPTION
**Pull request by yury-s**
Reference https://github.com/microsoft/playwright/issues/27792

### CODE REVIEW

File: packages/playwright-core/src/server/dispatchers/frameDispatcher.ts [hunk 1]
```diff
@@ -218,19 +216,8 @@ export class FrameDispatcher extends Dispatcher<Frame, channels.FrameChannel, Br
...
-    let { localPaths } = params;
-    if (!localPaths) {
-      if (!params.streams)
-        throw new Error('Neither localPaths nor streams is specified');
-      localPaths = params.streams.map(c => (c as WritableStreamDispatcher).path());
-    }
-    for (const p of localPaths)
-      assert(path.isAbsolute(p) && path.resolve(p) === p, 'Paths provided to localPaths must be absolute and fully resolved.');
-    return await this._frame.setInputFiles(metadata, params.selector, { localPaths }, params);
+    const inputFileItems = await prepareFilesForUpload(this._frame, params);
```
Comment (dgozman) [inline packages/playwright-core/src/server/dispatchers/frameDispatcher.ts:219]: I feel like this belongs to the `server`, not `dispatchers`. WDYT?

File: packages/playwright-core/src/client/elementHandle.ts [hunk 2]
```diff
@@ -322,7 +283,7 @@ export async function convertInputFiles(files: string | FilePayload | string[] |
       error += ', please write it to a file and pass its path instead.';
     throw new Error(error);
   }
-  return { files: payloads };
+  return { payloads };
```
Comment (dgozman) [inline packages/playwright-core/src/client/elementHandle.ts:286]: Shouldn't you remove the check on the line 282 as well? Perhaps include the removal of this method and protocol field in the same PR?

File: packages/playwright-core/src/server/dispatchers/frameDispatcher.ts (hunk 1 above)
Comment (yury-s) [reply packages/playwright-core/src/server/dispatchers/frameDispatcher.ts:219]: Moved.

File: packages/playwright-core/src/client/elementHandle.ts (hunk 2 above)
Comment (yury-s) [reply packages/playwright-core/src/client/elementHandle.ts:286]: Done.

**Review by dgozman — APPROVED** [review]

**Pull request merged**